common/src/main/java/com/wynntils/commands/ConfigCommand.java

```diff
@@ -192,6 +235,81 @@ private LiteralCommandNode<CommandSourceStack> buildReloadConfigNode() {
         return reloadConfigArgBuilder.build();
     }
 
+    private LiteralCommandNode<CommandSourceStack> buildOverlayGroupNode() {
+        LiteralArgumentBuilder<CommandSourceStack> overlayGroupArgBuilder = Commands.literal("overlaygroup");
+
+        overlayGroupArgBuilder.then(Commands.argument("feature", StringArgumentType.word())
+                .suggests(OVERLAY_GROUP_FEATURE_SUGGESTION_PROVIDER)
+                .then(Commands.argument("group", StringArgumentType.word())
+                        .suggests(OVERLAY_GROUP_SUGGESTION_PROVIDER)
+                        .then(Commands.literal("add").executes(this::addOverlayGroup))
+                        .then(Commands.literal("remove")
+                                .then(Commands.argument("id", StringArgumentType.word())
+                                        .suggests(OVERLAY_GROUP_REMOVE_ID_SUGGESTION_PROVIDER)
+                                        .executes(this::removeOverlayGroup)))));
+
+        return overlayGroupArgBuilder.build();
+    }
+
+    private int addOverlayGroup(CommandContext<CommandSourceStack> context) {
+        String featureName = context.getArgument("feature", String.class);
+        String groupName = context.getArgument("group", String.class);
+
+        OverlayGroupHolder overlayGroupHolder = getOverlayGroupHolderFromArguments(context, featureName, groupName);
+
+        if (overlayGroupHolder == null) return 0;
```

Comment:
Error message?

Reply:
Handled in the getter like every other method here.